src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classnames from 'classnames';
+import { px } from '../../utils';
```

Comment:
I believe your baseClassname will have the `px` prefix and you will not have to write your own here. If you look in the inspector you will see the `phillips-accordion` being applied twice.